src/ServiceBus/ServiceBus/help/Set-AzServiceBusNetworkRuleSet.md

```diff
@@ -0,0 +1,256 @@
+```
+
+### NetwrokruleSetInputObjectSet
+```
+Set-AzServiceBusNetworkRuleSet [-ResourceGroupName] <String> [-Name] <String>
+ [-InputObject] <PSNetworkRuleSetAttributes> [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+### NetworkRuleSetResourceIdParameterSet
+```
+Set-AzServiceBusNetworkRuleSet [-ResourceGroupName] <String> [-Name] <String> [-ResourceId] <String>
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Update the NetwrokruleSet of the given Namepsace in the current Azure subscription.
+
+## EXAMPLES
+
+### Example 1 
+```powershell
+PS C:\> $IpRules = @([Microsoft.Azure.Commands.ServiceBus.Models.PSNWRuleSetIpRulesAttributes] @{IpMask = "4.4.4.4";Action = "Allow"},[Microsoft.Azure.Commands.ServiceBus.Models.PSNWRuleSetIpRulesAttributes] @{IpMask = "3.3.3.3";Action = "Allow"})
+PS C:\> $VirtualNetworkRules = @([Microsoft.Azure.Commands.ServiceBus.Models.PSNWRuleSetVirtualNetworkRulesAttributes]@{Subnet=@{Id="/subscriptions/subscriptionId/resourcegroups/ResourceGroup/providers/Microsoft.Network/virtualNetworks/sbehvnettest1/subnets/default"};IgnoreMissingVnetServiceEndpoint=$True})
+PS C:\>  Set-AzServiceBusNetworkRuleSet -ResourceGroupName v-ajnavtest -Namespace ServiceBus-Namespace1-1375 -IPRules $IpRules -VirtualNtewrokRules $VirtualNetworkRules -DefaultAction "Allow" -Debug
+
+```
+
+Name                : default
+DefaultAction       : Allow
+Id                  : /subscriptions/subscriptionId/resourcegroups/ResourceGroup/providers/Microsoft.ServiceBus/namespaces/ServiceBus-Namespace-1122/networkRuleSets/default
+Type                : Microsoft.ServiceBus/Namespaces/NetworkRuleSet
+IpRules             : {4.4.4.4, Allow, 3.3.3.3, Allow}
+VirtualNetworkRules : {/subscriptions/subscriptionId/resourcegroups/ResourceGroup/providers/Microsoft.Network/virtualNetworks/sbehvnettest1/subnets/default, True}
+
+Update the NetworkRuleSet using Properties (ParameterSet = NetworkRuleSetPropertiesSet)
+
+### Example 2
+```powershell
+PS C:\> $getresult = Get-AzServiceBusNetworkRuleSet -ResourceGroupName v-ajnavtest -Namespace ServiceBus-Namespace1-1375
+PS C:\> Set-AzServiceBusNetworkRuleSet -ResourceGroupName v-ajnavtest -Namespace ServiceBus-Namespace1-1375 -InputObject $getresult
+```
+
+Name                : default
+DefaultAction       : Allow
+Id                  : /subscriptions/subscriptionId/resourcegroups/ResourceGroup/providers/Microsoft.ServiceBus/namespaces/ServiceBus-Namespace-1122/networkRuleSets/default
+Type                : Microsoft.ServiceBus/Namespaces/NetworkRuleSet
+IpRules             : {4.4.4.4, Allow, 3.3.3.3, Allow}
+VirtualNetworkRules : {/subscriptions/subscriptionId/resourcegroups/ResourceGroup/providers/Microsoft.Network/virtualNetworks/sbehvnettest1/subnets/default, True}
+
+Update the NetworkRuleSet using -IPRules and -VirtualNtewrokRules parameters
```

Comment:
@v-Ajnava same comment here about changing `-IPRules` and `-VirtualNtewrokRules` to their new names